bpffs/fs.go

```diff
@@ -8,7 +8,7 @@ import (
 const (
 	BPFFSPath = "/sys/fs/bpf"
 	// https://github.com/coreutils/coreutils/blob/v8.27/src/stat.c#L275
-	FsMagicBPFFS = 0xCAFE4A11
+	FsMagicBPFFS = int64(0xCAFE4A11)
```

Comment:
`0xCAFE4A11` fits on a `uint32` but not on a `int32`.

`syscall.Statfs_t.Type` can be a `int32 or a `int64` depending on the architecture.

I was wondering if `FsMagicBPFFS` could be declared as a constant of type `syscall.Statfs_t.Type` (which would be substituted to `int32` or `int64`) but I don't think that's possible with Go constants.

And 0xCAFE4A11 would need to be written as a negative value (−889304559) when it's `int32`, but a different negative value when it's `int64` (expending the signed bit `0xffffffff` on the most significant bits).

So I can only think of writing different implementations of `func IsMounted()` for different architectures.